k-es-grpc/src/main/kotlin/no/ks/kes/grpc/GrpcSubscriptionException.kt

```diff
@@ -0,0 +1,12 @@
+package no.ks.kes.grpc
+
+
+class GrpcSubscriptionException(val reason: GrpcSubscriptionReason, cause: Exception?) :
+    RuntimeException("Subscription was dropped. Reason: $reason", cause) {
```

Comment:
Unngå å kalle det dropped? Subscription failed? Error on subscription?

Reply:
Kan også sette cause til null default, så slipper man to konstruktører?

Reply:
Godt poeng :+1: